docs/azure.md

```diff
@@ -0,0 +1,116 @@
+# DeepSpeed with Azure
+
+This tutorial will help you get started running DeepSpeed on [Azure virtual
+machines](https://azure.microsoft.com/en-us/services/virtual-machines/). Support for
+[Azure ML](https://azure.microsoft.com/en-us/services/machine-learning/) will be coming
+soon!
+
+To help with launching Azure instances we suggest using the [Azure
+CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest). We have created
+several helper scripts to get you quickly started using DeepSpeed with Azure.
+ * Install Azure CLI on your local box: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli
+ * Alternatively you can use the Azure in-browser shell: https://shell.azure.com/
+
+## Create an SSH key
+Generate an SSH key that will be used across this tutorial to SSH into your VMs and
+between Docker containers. `ssh-keygen` is the recommended way of doing this. Our scripts
+assume your key is located inside the same directory as the Azure scripts.
+
+## Azure Config JSON
```

Comment:
I think it would be good to add a sentence or two telling users about the hardware that our sample configuration uses. We should also link to the SKU info page so pricing is clear before they follow the tutorial.